frontend/src/App.css

```diff
@@ -93,9 +93,35 @@ body {
   background-color: #0a2ea4;
 }
 
+.check-box {
+  transform: scale(1.5);
+  margin-right: 5%;
+}
 .todo-form {
```

Comment:
Shouldn't there be a space between these two different styles? The same for all other styles here.